README.md

```diff
@@ -72,11 +73,38 @@ Tha last step is to install the service and remove the downloaded files:
 
 ```bash
 /data/dbus-opendtu/install.sh
+/data/dbus-opendtu/restart.sh
 rm main.zip
 ```
 
 Check configuration after that - because the service is already installed and running. In case of wrong connection data (host, username, pwd) you will spam the log-file! Also, check to **set a** proper (minimal) **log level**
 
+### Update the code
+Just grap a copy of the main branch and copy the content to `/data/` e.g. `/data/dbus-opendtu`. The process will preserve your existing config ini. 
+After that call the `install.sh script.
+
+```bash
+wget -O main.zip https://github.com/henne49/dbus-opendtu/archive/refs/heads/main.zip
+unzip main.zip "dbus-opendtu-main/*" -d /data
+rm /data/dbus-opendtu/current.log*
+mv /data/dbus-opendtu-main/config.ini /data/dbus-opendtu-main/config.template
+cp -R /data/dbus-opendtu-main/* /data/dbus-opendtu
+rm -rf /data/dbus-opendtu-main/
+cp /data/dbus-opendtu/config.ini /data/dbus-opendtu/config.backup
```

Comment:
I sugget to do this copy earlier (maybe even befor the wget) - this will reduce the risk of it to be overwritten.

Reply:
that is changed now and the first steo